pkgs/top-level/python-packages.nix

```diff
@@ -27121,13 +27121,15 @@ in modules // {
     };
   };
 
-  udiskie = buildPythonPackage rec {
-    version = "1.4.8";
+  udiskie = buildPythonApplication rec {
```

Comment:
this is indeed an application. As such, this expression should be moved outside of `python-packages.nix` and put somewhere else in `nixpkgs`, since `python-packages.nix` is only intended for libraries or packages that need to be available for Python versions.